eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -9,8 +9,6 @@ jobs:
     - subset: coreclr
       include:
       - src/libraries/System.Private.CoreLib/*
-      - src/libraries/Native/Unix/System.Globalization.Native/*
```

Comment:
cc @safern - you mentioned Globalization in the issue I recently opened, asking about our zlib consumption.

Reply:
This is just pure cleanup as this will now be covered by `-src/native/*`?

Reply:
Yup, https://github.com/dotnet/runtime/pull/61958#discussion_r755328194, we previously added `src/native/*` path in https://github.com/dotnet/runtime/commit/565f3ee5ad68dad35305e9157e37fcdccc2aa2b6 (to trigger build of all related subsets when something changes under this directory).